policycoreutils/setsebool/setsebool.c

```diff
@@ -38,10 +38,7 @@ int main(int argc, char **argv)
 	if (argc < 2)
 		usage();
 
-	if (is_selinux_enabled() <= 0) {
-		fputs("setsebool:  SELinux is disabled.\n", stderr);
-		return 1;
-	}
+	reload = (is_selinux_enabled() > 0);
```

Comment:
Alternatively, we could just skip the call to semanage_set_reload() unless -N was specified, as we currently do in semodule and semanage/seobject.py, and then libsemanage would internally check is_selinux_enabled() itself to decide the default for reloading.